### DISCUSSION TITLE
Info

### DISCUSSION THREAD

**Discussion by t-rayne** [community/community#21236] [Other Feature Feedback, Questions, & Ideas]
Hey,  I was checking out your blog,  good job by the way very informative. I see your very knowledgable when it comes to Azure Active Directory and organizational structure so I was wondering if you had any advice you can offer pertaining to the removal of an unaffiliated personal pc from an unknown organization.

**Comment by codesandbax** [comment]
[](https://www.afsanhhydry12@gmail.com) #@codesandbox

**Reply by vk7kwb** [reply]
I am lost trying to get ARDUINO program working as it needs the Fdebrabander version of Arduino -Liquid Crystal-12c library to work 
other version tried not working .
Sketch from Silicon Chip 2020 WIDEBAND DIGITAL RF POWER METER .
Bryan W . bjwatson3@optusnet.com.au

**Reply by jfullstackdev** [reply]
do you use the TinkerCAD for Arduino ? maybe you can send the link here so I can check it too

**Comment by vk7kwb** [comment]
Hi No Tinker cad

I have a board from SILICON CHIP
and the  Sketch  form Silicon Chip 
I have tried various i2c libraries  none work 
as Silicon Chip stated it only works with the Fdebrabander  i2c
Sketch .
Bryan VK7KWB 

----- Original Message -----
From:
 "community/community"
***@***.***>

To:
"community/community" ***@***.***>
Cc:
"vk7kwb" ***@***.***>, "Comment"
***@***.***>
Sent:
Tue, 26 Jul 2022 23:31:43 -0700
Subject:
Re: [community/community] Info (Discussion #21236)

	do you use the TinkerCAD for Arduino ? maybe you can send the link
here so I can check it too 

	—
Reply to this email directly, view it on GitHub [1], or unsubscribe
[2].
You are receiving this because you commented.Message ID:
<community/community/repo-discussions/21236/comments/3257894
@
github
.
com>

-------------------------
Email sent using Optus Webmail

Links:
------
[1]
https://github.com/orgs/community/discussions/21236#discussioncomment-3257894
[2]
